src/java/picard/cmdline/CommandLineParser.java

```diff
@@ -340,6 +340,16 @@ public void usage(final PrintStream stream, final boolean printCommon) {
         }
     }
 
+    private String convertFromHtml(final String textToConvert) {
+
+        String regex1 = "<br />|<p>|</p>|<table>|</table >|<a href='(.*)'>(.*)</a>|<h4>|<pre>|</pre>|<ul>|</ul>|</li>|</tr>|<hr />";
+        String regex2 = "</h4>|<tbody>|</tbody>|<tr>|<th>";
```

Comment:
> can this be tested somehow? perhaps add an html doc (that uses all the tags you are looking for and some that you are not) and the expected output and make sure that they are the same?

If I had infinite time I would say yes, but in a time-constrained world I think it's overkill. I'm happy eyeballing a few docs and set the threshold at "do they look reasonable/readable?". If people find problems they'll be easy to fix quickly now that we have this set up.